kernel/packages/shared/atlas/sagas.ts

```diff
@@ -39,14 +36,10 @@ export function* atlasSaga(): any {
   yield fork(fetchDistricts)
   yield fork(fetchTiles)
 
-  yield takeEvery(SCENE_START, querySceneName)
   yield takeEvery(SCENE_LOAD, checkAndReportAround)
-  yield takeEvery(FETCH_NAME_FROM_SCENE_JSON, fetchName)
 
-  yield takeLatest(RENDERER_INITIALIZED, reportScenesAround)
```

Comment:
Could you explain a little why this is no longer needed?

Reply:
This is because now we aren't using the scenes lifecycle to fetch the scene data, as we need to take data from scenes that aren't even loaded. We just fetch them. So, with this decoupling the actions binding the lifecycle to the atlas saga were removed.

Reply:
It is not needed as the events that trigger the report of minimap information are wired to scene lifecycle events which are fired after the renderer is initialized.